src/SearchInput/SearchInput.tsx

```diff
@@ -186,7 +160,16 @@ const InputBox = styled.div<IShowIcon>`
     outlined ? `${theme.colors.outline}` : `${theme.colors.secondary}`};
 `
 
-const Input = styled.input<ISearchInput>`
+interface Input extends UsesOutline {
+  id: string
+  name: string
+  value: string
+  selected: boolean
+  onKeyUp: (e: React.FormEvent<HTMLInputElement>) => void
+  onChange: (e: React.FormEvent<HTMLInputElement>) => void
```

Comment:
Shouldn't those already be on `input` API? I don't think we need to declare them explicitly?